app/presenters/service_presenter.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class ServicePresenter < SimpleDelegator
+  def as_json(*)
+    {service: stringify_nil_values(filtered_values(__getobj__))}.merge(error_messages(__getobj__))
```

Comment:
Is there a better way to "pipe" functions in Ruby? cc: @guicassolato @hallelujah @Martouta

Reply:
Why do you use `__getobj__` here? You could use it in the private method ...